certification/pyxis/submit.go

```diff
@@ -37,36 +45,38 @@ func (p *pyxisClient) SubmitResults(ctx context.Context, certInput *Certificatio
 		}
 	}
 
-	dockerImageDigest := certImage.DockerImageDigest
```

Comment:
good catch, this was used in two places before, now only one. Smart to remove it.

Reply:
This actually needs to stay, I forgot we do not get any useful information back on conflicts, so we save this off first for calls to `GET`